### PR TITLE
fixed the outdated link in "Use a User Namespace With a Pod" page

### PR DESCRIPTION
**Pull request by aakashdeshamne**
Fixed the outdated links in the page "Use a User Namespace With a Pod" which is 
https://kubernetes.io/docs/tasks/configure-pod-container/user-namespaces.md

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/user-namespaces.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ this is true when user namespaces are used.
 * You need to exec commands in the host
 * You need to be able to exec into pods
 * You need to enable the `UserNamespacesSupport`
-  [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (Gauravpadam) [inline content/en/docs/tasks/configure-pod-container/user-namespaces.md:44]: Why this change?

File: content/en/docs/tasks/configure-pod-container/user-namespaces.md [hunk 2]
```diff
@@ -50,12 +50,12 @@ Only Kubernetes v1.25 through to v1.27 recognise `UserNamespacesStatelessPodsSup
 {{</ note >}}
 
 The cluster that you're using **must** include at least one node that meets the
-[requirements](/docs/concepts/workloads/pods/user-namespaces/#before-you-begin)
+[requirements]( https://kubernetes.io/docs/concepts/workloads/pods/user-namespaces/)
```
Comment (Gauravpadam) [inline content/en/docs/tasks/configure-pod-container/user-namespaces.md:53]: Why this change?

File: content/en/docs/tasks/configure-pod-container/user-namespaces.md [hunk 3]
```diff
@@ -50,12 +50,12 @@ Only Kubernetes v1.25 through to v1.27 recognise `UserNamespacesStatelessPodsSup
...
 for using user namespaces with Pods.
 
 If you have a mixture of nodes and only some of the nodes provide user namespace support for
 Pods, you also need to ensure that the user namespace Pods are
-[scheduled](/docs/concepts/scheduling-eviction/assign-pod-node/) to suitable nodes.
+[scheduled](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/) to suitable nodes.
```
Comment (Gauravpadam) [inline content/en/docs/tasks/configure-pod-container/user-namespaces.md:58]: Why this change?

**Review by Gauravpadam — COMMENTED** [review]
We address our resources using relative urls

**Comment by Gauravpadam** [comment]
Welcome @aakashdeshamne!

If you wish to contribute, I'd suggest looking into our [active issues](https://github.com/kubernetes/website/issues?q=is%3Aissue+is%3Aopen+label%3Atriage%2Faccepted)

File: content/en/docs/tasks/configure-pod-container/user-namespaces.md (hunk 3 above)
Comment (aakashdeshamne) [reply content/en/docs/tasks/configure-pod-container/user-namespaces.md:58]: it mention in issue #44573  that they are outdated need to fix

Comment (aakashdeshamne) [reply content/en/docs/tasks/configure-pod-container/user-namespaces.md:58]: can you please explain me what exactly to change there

Comment (aakashdeshamne) [reply content/en/docs/tasks/configure-pod-container/user-namespaces.md:58]: > Why this change?
it was mention in issue #44573 that they are outdated and need to fix can you please me what exactly to change

**Comment by aakashdeshamne** [comment]
> Welcome @aakashdeshamne!
> 
> If you wish to contribute, I'd suggest looking into our [active issues](https://github.com/kubernetes/website/issues?q=is%3Aissue+is%3Aopen+label%3Atriage%2Faccepted)

@Gauravpadam can you please explain why it is mention in issue https://github.com/kubernetes/website/issues/44573 that they are outdated and need to fix can you please me what exactly to change

**Comment by Gauravpadam** [comment]
https://github.com/kubernetes/website/pull/44571 had fixed that issue; so there was essentially nothing broken anymore

I've closed the issue as completed
